pkg/topo/topo.go

```diff
@@ -60,8 +59,8 @@ func NewClient(opts ...Option) (Client, error) {
 	}
 
 	dialOpts := []grpc.DialOption{
-		grpc.WithStreamInterceptor(southbound.RetryingStreamClientInterceptor(100 * time.Millisecond)),
-		grpc.WithUnaryInterceptor(southbound.RetryingUnaryClientInterceptor()),
+		grpc.WithUnaryInterceptor(retry.RetryingUnaryClientInterceptor(retry.WithRetryOn())),
+		grpc.WithStreamInterceptor(retry.RetryingStreamClientInterceptor(retry.WithRetryOn())),
```

Comment:
This is actually telling it to retry on nothing (never). Just remove the retry option.